test/functional/basic/gesture-e2e-specs.js

```diff
@@ -41,15 +40,27 @@ describe('XCUITestDriver - gestures', function () {
           await el.click();
         });
       });
+
+      async function exitModal (name) {
+        // should exist, will throw error if it doesn't
+        let els = await driver.elementsByAccessibilityId(name);
```

Comment:
don't we have a dedicated method to find a single element and throw an exception otherwise?